doc/code/Apps.md

```diff
@@ -35,18 +35,20 @@ that will call the method `Refresh()` periodically.
 
 ## App types
 
-There are basically 2 types of applications : **system** apps and **user** apps.
+There are basically 3 types of applications : **system** apps and **user** apps and **watchfaces**.
```

Comment:
In documentation we opted to use `watch faces` instead of `watchfaces`. I'd like to keep it consistent. Only if it is a named type, then I'd keep it as one word

```suggestion
There are basically 3 types of applications : **system** apps and **user** apps and **watch faces**.
```